kotlin-reference-server/src/main/kotlin/org/stellar/ReferenceServer.kt

```diff
@@ -0,0 +1,70 @@
+package org.stellar
```

Comment:
Hey @Ifropc , I was not specific enough. Can we use the package name: `org.stellar.reference`?